src/Modules/MotdModule/Models/MotdResponse.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace EvoSC.Modules.Official.MotdModule.Models;
+
+public class ResponseData
+{
+    public int id { get; set; }
+    public string message { get; set; }
+    public string server { get; set; }
+}
+
+public class MotdResponse
+{
+    public List<ResponseData> data { get; set; }
```

Comment:
```suggestion
    public List<ResponseData> Data { get; set; }
```

Also add doc comment.